canon-docs/public/styles.css

```diff
@@ -0,0 +1,2 @@
+/*! modern-normalize v3.0.1 | MIT License | https://github.com/sindresorhus/modern-normalize */
```

Comment:
is this minified on purpose?

Reply:
+1 to keeping styles non-minified to simplify readability in editors etc. These are less important I guess but especially `packages/canon/css/styles.css` in the published package should not be minified.

Reply:
It'll be minified in the final production build anyway though when built in production mode